### ISSUE TITLE
TorchHubRegistery: the out-of-the-box registery for Rikai

### ISSUE THREAD

**Issue by darcy-shen** [eto-ai/rikai#368]
Rikai should provide nice default pre/post preprocessors for some of them. In the way, anyone can get themselves started without pain using Rikai SQL.

https://pytorch.org/hub/ultralytics_yolov5/
``` sql
CREATE MODEL yolov5
OPTIONS (min_confidence=0.3, device="gpu", batch_size=32)
USING "torchhub:///ultralytics/yolov5/yolov5s";
```

https://pytorch.org/hub/nvidia_deeplearningexamples_resnet50/
``` sql
CREATE MODEL yolov5
OPTIONS (min_confidence=0.3, device="gpu", batch_size=32)
USING "torchhub:///NVIDIA/DeepLearningExamples:torchhub/nvidia_resnet50";
```

**Comment by darcy-shen** [comment]
With TorchHubRegistery, our documentation can be improved using runnable SQL: https://github.com/eto-ai/rikai/blob/master/docs/sqlml.rst

**Comment by darcy-shen** [comment]
with #391 merged, I will move to work on #356

**Comment by eddyxu** [comment]
close this since #391 is merged
